README.md

```diff
@@ -1 +1,58 @@
-# Awesome-books-ES6
\ No newline at end of file
+# Microverse Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Awesome books with ES6
+
+> This is a solo project being built to show the usage of html and javascript. its a book application were you are able to add and remove book to a list
+
+Our goal here is to Build a book application with add and remove feature from scratch using JavaScripts.
+
+## Features implemented so far are:
+
+Responsive HTML page 
+Add book feature
+Remove book feature
+Saving current state of book list to local storage
+The classes and methods are built using modules and arrow function respectively. 
+The date feature was built using luxon library
+
+## Built With
+
+>- HTML
+>- Javascript
+
+## Getting Started
+> To get a local copy, follow the following steps
+
+Prerequisites
+>- VS code
+>- Git Bash
+
+Setup
+>- Git clone `git@github.com:V0pe/Awesome-books-ES6.git`
+
+
+## Author
+
+👤 **Victor Damola**
+
+>- GitHub: [@V0pe](https://github.com/V0pe)
+>- Twitter: [@de_vope_](https://twitter.com/de_vope)
+>- LinkedIn: [Victor Damola](https://linkedin.com/in/victor-damola-aderibigbe-27931ab0)
+
+
+## 🤝 Contributing
+
+>Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+>Give a ⭐️ if you like this project!
+
+## Acknowledgement
+
+>Greatful to @microverseinc for putting this project together
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The `MIT` link is broken. Please let's fix it. This will render our README file professional.

Reply:
@FrederickMih  You are right. I should have added it. I have done it now. Thanks _